cypress/e2e/candidate.feature

```diff
@@ -0,0 +1,18 @@
+Feature: Candidate Interview Result Verification Pass/Fail
+    Scenario: The Admin should successfully transition the candidate's status to  Interview Passed
+        Given  Admin login
+        Given Created Employee
+        Given Created job
+        Given Created Vacancy
+        Given Created Candidate
```

Comment:
Should put these       
      ` Given  Admin login`
        `Given Created Employee`
      `  Given Created job`
        `Given Created Vacancy`
       ` Given Created Candidate` in before hook